cw/cw305/capture.py

```diff
@@ -37,15 +37,16 @@ class ScopeType(str, Enum):
 opt_num_traces = typer.Option(None, help="Number of traces to capture.")
 opt_plot_traces = typer.Option(None, help="Number of traces to plot.")
 opt_scope_type = typer.Option(ScopeType.cw, help=("Scope type"))
+opt_ciphertexts_gen = typer.Option(False, help=("Generate all ciphertexts for batch capture."))
```

Comment:
This would better be called `opt_ciphertexts_store` because the ciphertexts are generated anyway on the target. It's the result of the encryption.

Reply:
Then, where the argument is used inside `aes_fvsr_key_batch()` there should be commend saying by default we don't store the ciphertexts as generating them on the host as well as transferring them over from the target substantially reduces capture performance. It should therefore only be enabled if absolutely needed.

Reply:
Perfect, thanks!